python/tidy/servo_tidy/tidy.py

```diff
@@ -706,11 +737,16 @@ def collect_errors_for_files(files_to_check, checking_functions, line_checking_f
             continue
         with open(filename, "r") as f:
             contents = f.read()
+            lines = contents.splitlines(True)
+            if filename.endswith(os.path.join(os.sep, "servo-tidy.toml")):
```

Comment:
This may not be the correct place to check the config file, since that's probably the first thing that we should be doing. Let's move it to `scan`, have a separate variable binding `config_errors` for this thing, and chain with the other iterators in such a way that this is done first.